CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.1.0](https://github.com/voxpupuli/beaker-google/tree/1.1.0)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-google/compare/1.0.0...1.1.0)
+
+**Implemented enhancements:**
+- Remove timestamp from VM and firewall name [\#46](https://github.com/voxpupuli/beaker-google/pull/46) ([jaevans](https://github.com/jaevans))
```

Comment:
Generate the changelog using:

```
bundle exec rake release:prepare
```

Reply:
I followed the directions at https://voxpupuli.org/docs/releasing_gem/, are those not the correct directions? Is there another place with more up-to-date or better directions for new maintainers?

Reply:
https://voxpupuli.org/docs/releasing_version/#preparing-a-release is the correct up2date documentation